packages/hub/main.js

```diff
@@ -67,6 +67,16 @@ async function startIndexing(environment, container) {
     let searchers = await container.lookup(`hub:searchers`);
     let models = await (await container.lookup('config:initial-models'))();
 
+    let branch = 'master';
+    let writers = container.lookup('hub:writers');
+
+    // write cardstack-files to data source
+    for (let model of models) {
+      if (model.readable) {
+        await writers.createBinary(branch, Session.INTERNAL_PRIVILEGED, 'cardstack-files', model);
+      }
+    }
+
```

Comment:
Why are we writing out models in `startIndexing`? This seems like a data destruction bug.

The ephemeral data source is the only one that _always_ loads seed data automatically. The others only load seed data if you run the hub:seed command. But they will go through `startIndexing` and had better not overwrite their contents with the seed models when they do.